beamer/agent/commands.py

```diff
@@ -41,10 +41,16 @@ def _sigint_handler(agent: Agent) -> None:
     "--account-password", type=str, prompt=False, help="The password needed to unlock the account."
 )
 @click.option(
-    "--deployment-dir",
+    "--artifact-dir",
```

Comment:
Nitpick: if we are to be consistent with the `beamer deploy` command, this should be `--artifacts-dir`: